configure.ac

```diff
@@ -77,6 +77,12 @@ AC_SEARCH_LIBS([rpmsqSetInterruptSafety], [rpmio],
   AC_DEFINE([BUILDOPT_HAVE_RPMSQ_SET_INTERRUPT_SAFETY], 0, [Set to 1 if we have interrupt safety API])
 )
 
+# rpmfiles was made public in rpm >= 4.12, el7 is still on 4.11
+AC_SEARCH_LIBS([rpmfilesNew], [rpm],
+  AC_DEFINE([BUILDOPT_HAVE_RPMFILES], 1, [Set to 1 if we have rpmfiles API]),
+  AC_DEFINE([BUILDOPT_HAVE_RPMFILES], 0, [Set to 1 if we have rpmfiles API])
```

Comment:
Thanks for looking though!  This is indeed just autoconf being crazy, but it could easily have been a bug.